src/babel.js

```diff
@@ -297,7 +297,8 @@ export default function ({types: t}) {
           const [id, css, loc] = state.styles.shift()
 
           if (isGlobal) {
-            path.replaceWith(makeStyledJsxTag(id, css.source || css, css.modified))
+            const transformedCss = transform(null, (css.source || css))
```

Comment:
nothing wrong with this but you could also move `let transfromedCss` (L306) to L298